extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -73,6 +77,12 @@
   border-bottom-right-radius: 50% !important;
 }
 
+[type="emoji"] .i-amphtml-story-interactive-slider-bubble-content {
+  height: 100% !important;
+  width: 100% !important;
+  font-size: 1.5em !important;
+}
+
```

Comment:
If we just want to change the font size of the emoji, we could use the first-line selector, like:
```
[type="emoji"] .i-amphtml-story-interactive-slider-bubble:first-line {
  font-size: 1.5em !important;
}
```